src/dataProvider/model/MediaComposition.js

```diff
@@ -197,13 +198,14 @@ class MediaComposition {
       ),
       blockReason: this.getMainChapter().blockReason,
       blockedSegments: this.getMainBlockedSegments(),
+      chapterImageUrl: this.getMainChapterImageUrl(),
```

Comment:
Since the function is called `getMainChapterImageUrl` I would preserve the naming in this property and called it `mainChapterImageUrl` as well.

Reply:
This means, that all properties grabbed from `getMainChapter` should keep the same logic, or am I missing something ?

Reply:
Totally agree, give me one second.